docs/readthedocs/source/doc/Orca/Tutorial/k8s.md

```diff
@@ -274,49 +276,45 @@ If your program depends on a nested directory of Python files, you are recommend
 
 
 ---
-## 6. Run Jobs on K8s
-In the following part, we will illustrate four ways to submit and run BigDL Orca applications on K8s.
+## 7. Run Jobs on K8s
+In the following part, we will illustrate three ways to submit and run BigDL Orca applications on K8s.
 
 * Use `python` command
 * Use `spark-submit`
-* Use Kubernetes Deployment (with Conda Archive)
-* Use Kubernetes Deployment (with Integrated Image)
+* Use Kubernetes Deployment
 
 You can choose one of them based on your preference or cluster settings.
 
 We provide the running command for the [Fashion-MNIST example](https://github.com/intel-analytics/BigDL/blob/main/python/orca/tutorial/pytorch/FashionMNIST/) in the __Client Container__ in this section.
 
-### 6.1 Use `python` command
+### 7.1 Use `python` command
 This is the easiest and most recommended way to run BigDL Orca on K8s as a normal Python program.
 
 See [here](#init-orca-context) for the runtime configurations.
 
-#### 6.1.1 K8s-Client
+#### 7.1.1 K8s-Client
 Run the example with the following command by setting the cluster_mode to "k8s-client":
 ```bash
 python train.py --cluster_mode k8s-client --data_dir /bigdl/nfsdata/dataset
 ```
 
 
-#### 6.1.2 K8s-Cluster
-Before running the example on `k8s-cluster` mode, you should:
-* In the __Client Container__:
+#### 7.1.2 K8s-Cluster
+Before running the example on `k8s-cluster` mode in the __Client Container__, you should:
 
-Pack the current activate conda environment to an archive:
-```bash
-conda pack -o environment.tar.gz
-```
-
-* On the __Develop Node__:
```

Comment:
changed to perform this in the client container.
On the develop node, the nfs path is not /bigdl/nfsdata